js/details.js

```diff
@@ -10,36 +10,17 @@ function show_probe( e, text ) {
 		})
 }
 
-function show_details( properties ) {
+function show_details( properties, executing ) {
```

Comment:
Identifier 'show_details' is not in camel case.<br>'show_details' is defined but never used.